### ISSUE TITLE
Dynamically changing icon size doesn't affect font size

### ISSUE THREAD

**Issue by Sandsten** [Sitebase/react-avatar#167]
In this example I'm changing the avatar size depending on the variable "iconSize". The text with the initials always stay in the same size in which they were first rendered, even when dynamically changing between `size={50}` and `size={100}`. Shouldn't the text also change in this case?

```
      if(iconSize == "small") {
        return <Avatar name={`${user.firstName} ${user.lastName}`} size={50} round="50%" />;
      } else {
        return <Avatar name={`${user.firstName} ${user.lastName}`} size={100} round="50%" />;
      }
```

**Comment by JorgenEvens** [comment]
Hi @Sandsten, I'll be looking into this over the weekend.

**Comment by JorgenEvens** [comment]
Hi @Sandsten, sorry it took me longer than anticipated before I could have a decent look. The good news however is that a fix was included in the [v3.7.0](https://github.com/Sitebase/react-avatar/milestone/33?closed=1) release!
